### DISCUSSION TITLE
Too many redirects after logging in

### DISCUSSION THREAD

**Discussion by FedericoGaglio** [keycloak/keycloak#17647] [Q&A]
Hi all, I'm having some problem with keycloak, configured to be able to use the reverse proxy (edge). 
The configuration I'm currently using is the following (put as a snippet in the input of my keycloak deployment ingress on kubernetes):

location /* {
        proxy_set_header    Host                $host;
        proxy_set_header    X-Real-IP           $remote_addr;
        proxy_set_header    X-Forwarded-For     $proxy_add_x_forwarded_for;
        proxy_set_header    X-Forwarded-Host    $host;
        proxy_set_header    X-Forwarded-Server  $host;
        proxy_set_header    X-Forwarded-Port    $server_port;
        proxy_set_header    X-Forwarded-Proto   $scheme;
        proxy_pass http://[MY-HOST];
    }

Instead as variables inside my doployment I set:

- name: KC_PROXY
          value: edge
 - name: PROXY_ADDRESS_FORWARDING
          value: "true"
(regarding the configuration of the reverse proxy).

Problem: I can successfully access the keycloak main page managing to go to the administrator console and managing to view the login form... but once I enter the credentials and send it, I get a TOO MANY REDIRECT error (on this url : realms/master/login-actions/authenticate?client_id=security-admin-console&tab_id=[TAB-ID]).

Can you kindly give me a hand. Thank you :) .

**Comment by durgadas-kamath** [comment]
Any solution here ? @FedericoGaglio - you managed to find solution ?

**Comment by tirelibirefe** [comment]
maybe you should share all oauth2-proxy config file to get a better respond.
